src/Integration/Telemetry/TelemetryHelper.cs

```diff
@@ -80,7 +82,22 @@ public static bool IsSonarCloud(Uri sonarqubeUri)
                 InstallDate = telemetryData.InstallationDate,
                 Analyses = telemetryData.Analyses,
                 ShowHotspot = telemetryData.ShowHotspot,
-                TaintVulnerabilities = telemetryData.TaintVulnerabilities,
+                TaintVulnerabilities = telemetryData.TaintVulnerabilities
+            };
+        }
+
+        private static IdeVersionInformation GetVsVersion(IVsVersion vsVersion)
+        {
+            if (vsVersion == null)
+            {
+                return null;
+            }
+
+            return new IdeVersionInformation
+            {
+                ProductName = vsVersion.ProductName,
+                ProductBuildVersion = vsVersion.ProductVersion,
+                ProductDisplayVersion = vsVersion.ProductDisplayVersion
```

Comment:
Are these the same field names as in other instance of SL?

If not, I think `ProductBuildVersion` would be better named `InstallationVersion` so it's easier to map back to the "official" VS versioning, and also because it isn't a build version.

Given the choice, I'd call the telemetry fields `DisplayName`, `InstallationVersion` and `DisplayVersion`.

Reply:
... and rename the `IVsVersion` properties to match.